content/data-reporting/reports-api/index.mdx

```diff
@@ -95,7 +95,59 @@ curl "https://api.livechatinc.com/v3.2/reports/chats/agents_chatting_duration?fr
 </Sample>
 </CodeSample>
 
-<CodeResponse type="reports" version="v3.3" json="agentsChattingDuration"/>
+<CodeResponse type="reports" version="v3.2" json="agentsChattingDuration"/>
+
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+### Tags
+
+Shows distribution of tags for chats.
+
+#### Specifics
+
+|                     |                                                       |
+| ------------------- | ----------------------------------------------------- |
+| **Method URL**      | `https://api.livechatinc.com/v3.2/reports/chats/tags` |
+| **HTTP method**     | GET                                                   |
+| **Required scopes** | -                                                     |
+
+#### Request
+
+| Query String          | Required | Notes                                                                                                                                                                   |
+| --------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                  | yes      | Date in the RFC3339 format                                                                                                                                              |
+| `from`                | yes      | Date in the RFC3339 format                                                                                                                                              |
+| `agents`              | no       | Agent emails separated by a comma; if not specified, returns the data for all agents within the license.                                                                |
+| `groups`              | no       | Group IDs separated by a comma                                                                                                                                          |
+| `names`               | no       | Names of tags separated by a comma; when tags=:without: you will get total number of chats without tags, when tags=:with: you will get total number of chats with tags. |
+| `distribution`        | no       | Possible values: `hour`, `day-hours`, `day`, `month`, `year`; defaults to `day`                                                                                         |
```

Comment:
What's day-hours?

Reply:
hour distribution groups by date and hour so keys in the result are like `2021-01-14 15:00`
on the other hand day-hours groups just by hour so keys in the result are like `15:00`